demo/datasource/source.py

```diff
@@ -124,7 +129,7 @@ def raw_train_data_generator(self):
         return iter(
             self._iter_rows(
                 query_reader=reader(self.train_queries_filepath, self.words),
-                intent_reader=reader(self.train_intent_filepath, self.intent),
+                intent_reader=reader(self.train_intent_filepath, self.intents),
```

Comment:
I believe this fix is already in. Please rebase your PR onto a recent master and check if it's still needed.

Reply:
This has indeed been fixed since this PR was made.